flycheck.el

```diff
@@ -5605,6 +5605,17 @@ Relative paths are relative to the file being checked."
   :safe #'flycheck-string-list-p
   :package-version '(flycheck . "0.18"))
 
+(flycheck-def-option-var flycheck-dmd-flags nil d-dmd
+  "A list of string import directories for dmd.
+
+The value of this variable is a list of strings, where each
+string is a compiler flag to add when calling dmd.
+Relative paths are relative to the file being checked."
+  :type '(repeat (directory :tag "Compiler flags"))
```

Comment:
`directory` is the wrong type.  `string` would be appropriate.  Also, the `tag` would be singular, i.e. `Compiler flag`, because it denotes a single element in the list.